.github/workflows/test.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         if:
           github.repository == 'psf/black' && matrix.os == 'ubuntu-latest' &&
           !startsWith(matrix.python-version, 'pypy')
-        uses: AndreMiras/coveralls-python-action@v20201129
+        uses: AndreMiras/coveralls-python-action@develop
```

Comment:
We should pin to a specific version for security.

Reply:
Yeah, I'd also prefer to depend on dependabot to update this ... Is there a feature you needed in develop tag / branch?

Reply:
Tags are mutable, if we care about security don't we need a hash?

Reply:
Make sense, updated to commit id.